recipes/hkl/COPYING

```diff
@@ -0,0 +1,675 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Why do you need this file here?

Reply:
To make sure it's packaged. Is it a bad practice?

Reply:
Yes, it's a bad practice. See docs on how to include the license file from source.

Reply:
OK, I'll remove it from here. So, the assumption is that the license file should be present in the source distribution conda pulls during the building process and then this file is packaged, not the one which is provided, right?

Reply:
I was misled by many feedstocks having a license file in their `recipe` dirs. That's easier to maintain if it's not included here 😃